memory/libhugetlbfs.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+
+import os
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils import build
+from avocado.utils import kernel
+from avocado.utils import memory
+from avocado.utils import git
+from avocado.utils.software_manager import SoftwareManager
+from avocado.utils import distro
+
+
+class libhugetlbfs(Test):
+    def setUp(self):
+        # Check for basic utilities
+        sm = SoftwareManager()
+        detected_distro = distro.detect()
+        if not sm.check_installed("gcc") and not sm.install("gcc"):
+            self.error('Gcc is needed for the test to be run')
+        if not sm.check_installed("make") and not sm.install("make"):
+            self.error('make is needed for the test to be run')
+        if detected_distro.name == "SuSE":
+            if not sm.check_installed("git-core") and not sm.install("git-"
+                                                                     "core"):
+                self.error('git is needed for the test to be run')
+        else:
+            if not sm.check_installed("git") and not sm.install("git"):
+                self.error('git is needed for the test to be run')
+
+        kernel.check_version("2.6.16")
+        op = process.system_output('find /usr/lib*/ -name "libpthread.a"',
+                                   verbose=False, shell=True)
+        if 'libpthread.a' not in op:
+            self.error("libpthread.a is required")
+
+        # Get arguments:
+        self.hugetlbfs_dir = self.params.get('hugetlbfs_dir', default=None)
+        pages_requested = int(self.params.get('pages_requested',
```

Comment:
Oh, if you remove the quotation marks you don't need the `int()`:
~~~
pages_requested = self.params.get('pages_requested', default=20)
~~~